.gitignore

```diff
@@ -54,8 +55,6 @@ systest/
 jmh-benchmarks/generated
 jmh-benchmarks/src/main/generated
 **/.jqwik-database
-**/src/generated
-**/src/generated-test
```

Comment:
Removing these ignores makes it more likely for people to accidentally commit the generated code, right?

WDYT about moving the directories but leaving the ignores in place for some time? If we merged my `clean` PR, that could give a window in which developers can clean up the old directories without ever thinking about them, and later we can finally remove the ignores and the extra `clean`.

Reply:
The reason I wanted to remove it was so we could do a simple `git clean` to remove the files from the old location. However, not ignoring them does make it more likely for someone to commit them. 

I think your plan makes sense. Basically if we move the files to "build", but keep the additional clean logic you've added, `./gradlew clean` should clear out both locations.

Reply:
+1 to @gharris1727 to leave the ignores in place. Otherwise, people could commit the generated code unexpectedly when testing 3.x branches and trunk branch.